storage/postgres/postgres-db.js

```diff
@@ -0,0 +1,63 @@
+'use strict';
+
+const config = require('../../config');
+const pg = require('pg');
```

Comment:
pg is the default postgre package I've seen used
but need to research around to make sure it's the best choice and what implementation to use

https://node-postgres.com/api/pool